saged/tissue_split.py

```diff
@@ -0,0 +1,230 @@
+"""
+This experiment determines whether models pretrained on data from different tissues than they
```

Comment:
Just so I understand - before (e.g. in #52), you were pretraining on data from a mixture of tissues including the one you were trying to predict expression in? And now you're pretraining on one tissue (or a mixture of tissues not including the holdout tissue), and predicting on a different one?

Reply:
Correct, the pretraining data is a different set of studies with different samples than the training and validation sets. The train/val sets have the same tissues, but are still split by study